src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,141 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.exception.InvalidUserException;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static RegistrationService registrationService;
+    private static User defaultUser;
+    private static final int MIN_AGE = 18;
+    private static final String VALID_PASSWORD = "123456";
+    private static final String DEFAULT_LOGIN = "testUser";
+
+    @BeforeAll
+    static void initializeService() {
+        registrationService = new RegistrationServiceImpl();
+    }
+
+    @BeforeEach
+    void initialUser() {
+        defaultUser = new User(DEFAULT_LOGIN, VALID_PASSWORD, MIN_AGE);
+    }
+
+    @AfterEach
+    void tearDown() {
+        Storage.people.clear();
+    }
+
+    @Test
+    void register_CorrectUser_Ok() {
```

Comment:
```suggestion
    void register_correctUser_ok() {
```
the same for other methdos